src/components/Calculator.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Please don't disable eslint rules. This is one of the basic requirements.

Reply:
Each component should be in its separate file so that you can re-use any component in the future, it's a bad practice to add multiple components in one file

please keep in mind that in W3School, they only show examples in one block of code for readability, but they tell you to use separate files as well 👇
![image](https://user-images.githubusercontent.com/11758151/129842284-eda70581-a7b9-4386-af9b-b5ef713852e8.png)